common/util/goal_distance_calculator/src/main.cpp

```diff
@@ -0,0 +1,27 @@
+// Copyright 2020 Tier IV, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <goal_distance_calculator/goal_distance_calculator_node.hpp>
+#include <rclcpp/rclcpp.hpp>
+
+#include <memory>
```

Comment:
Place cpp system header ahead of other headers.

Reply:
fixed at b0934a6